### ISSUE TITLE
1.14 version doesnt work

### ISSUE THREAD

**Issue by zxcdevlop** [Prospector/ModMenu#394]
when i launch fabric 1.14 with modmenu and I click on "mods" Minecraft just crashes, anyone know a solution?

**Comment by Prospector** [comment]
While 1.14 is pretty old, if it does indeed crash I would like to fix it. Could you upload a crash log?

**Comment by zxcdevlop** [comment]
i think this is the one:
```
---- Minecraft Crash Report ----
// Who set us up the TNT?

Time: 3/5/22 3:09 PM
Description: mouseClicked event handler

java.lang.NullPointerException: mouseClicked event handler
	at io.github.prospector.modmenu.gui.ModListWidget.filter(ModListWidget.java:133)
	at io.github.prospector.modmenu.gui.ModListWidget.filter(ModListWidget.java:117)
	at io.github.prospector.modmenu.gui.ModListWidget.<init>(ModListWidget.java:45)
	at io.github.prospector.modmenu.gui.ModListScreen.init(ModListScreen.java:88)
	at net.minecraft.class_437.init(class_437.java:330)
	at net.minecraft.class_310.method_1507(class_310.java:830)
	at io.github.prospector.modmenu.gui.ModMenuButtonWidget.lambda$new$0(ModMenuButtonWidget.java:9)
	at net.minecraft.class_4185.onPress(class_4185.java:18)
	at net.minecraft.class_4264.onClick(class_4264.java:15)
	at net.minecraft.class_339.mouseClicked(class_339.java:154)
	at net.minecraft.class_4069.mouseClicked(class_4069.java:27)
	at net.minecraft.class_442.mouseClicked(class_442.java:320)
	at net.minecraft.class_312.method_1611(class_312.java:86)
	at net.minecraft.class_437.wrapScreenError(class_437.java:441)
	at net.minecraft.class_312.method_1601(class_312.java:86)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3050)
	at net.minecraft.class_1041.method_16001(class_1041.java:408)
	at net.minecraft.class_1041.method_15998(class_1041.java:279)
	at net.minecraft.class_310.method_15994(class_310.java:1023)
	at net.minecraft.class_310.method_1523(class_310.java:976)
	at net.minecraft.class_310.method_1514(class_310.java:410)
	at net.minecraft.client.main.Main.main(Main.java:155)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at io.github.prospector.modmenu.gui.ModListWidget.filter(ModListWidget.java:133)
	at io.github.prospector.modmenu.gui.ModListWidget.filter(ModListWidget.java:117)
	at io.github.prospector.modmenu.gui.ModListWidget.<init>(ModListWidget.java:45)
	at io.github.prospector.modmenu.gui.ModListScreen.init(ModListScreen.java:88)
	at net.minecraft.class_437.init(class_437.java:330)
	at net.minecraft.class_310.method_1507(class_310.java:830)
	at io.github.prospector.modmenu.gui.ModMenuButtonWidget.lambda$new$0(ModMenuButtonWidget.java:9)
	at net.minecraft.class_4185.onPress(class_4185.java:18)
	at net.minecraft.class_4264.onClick(class_4264.java:15)
	at net.minecraft.class_339.mouseClicked(class_339.java:154)
	at net.minecraft.class_4069.mouseClicked(class_4069.java:27)
	at net.minecraft.class_442.mouseClicked(class_442.java:320)
	at net.minecraft.class_312.method_1611(class_312.java:86)

-- Affected screen --
Details:
	Screen name: net.minecraft.class_442
Stacktrace:
	at net.minecraft.class_437.wrapScreenError(class_437.java:441)
	at net.minecraft.class_312.method_1601(class_312.java:86)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3050)
	at net.minecraft.class_1041.method_16001(class_1041.java:408)
	at net.minecraft.class_1041.method_15998(class_1041.java:279)
	at net.minecraft.class_310.method_15994(class_310.java:1023)
	at net.minecraft.class_310.method_1523(class_310.java:976)
	at net.minecraft.class_310.method_1514(class_310.java:410)
	at net.minecraft.client.main.Main.main(Main.java:155)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.14.4
	Minecraft Version ID: 1.14.4
	Operating System: Mac OS X (x86_64) version 10.16
	Java Version: 1.8.0_74, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1590141512 bytes (1516 MB) / 2113929216 bytes (2016 MB) up to 2113929216 bytes (2016 MB)
	CPUs: 4
	JVM Flags: 13 total; -XX:+UnlockExperimentalVMOptions -XX:-UseParallelGC -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:+ParallelRefProcEnabled -Xms2000M -Xmx2000M -Xss1M
	Fabric Mods: 
		fabric: Fabric API 0.28.5+1.14
		fabric-api-base: Fabric API Base 0.1.2+de26dc4942
		fabric-biomes-v1: Fabric Biomes (v1) 0.1.2+4ea4772942
		fabric-command-api-v1: Fabric Command API (v1) 1.0.2+4ea4772942
		fabric-commands-v0: Fabric Commands (v0) 0.2.1+4ea4772942
		fabric-containers-v0: Fabric Containers (v0) 0.1.4+4ea4772942
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.1.3+4ea4772942
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+4ea4772942
		fabric-dimensions-v1: fabric-dimensions-v1 0.1.2+4ea4772942
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.1+4ea4772942
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+4ea4772942
		fabric-item-api-v1: Fabric Item API (v1) 1.1.1+4ea4772942
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.1+4ea4772942
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.2+4ea4772942
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.1+4ea4772942
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.1+4ea4772942
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 0.1.2+4ea4772942
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+4ea4772942
		fabric-models-v0: Fabric Models (v0) 0.1.1+4ea4772942
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.1+4ea4772942
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.2+4ea4772942
		fabric-networking-v0: Fabric Networking (v0) 0.2.1+4ea4772942
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.0.1+4ea4772942
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.3.1+4ea4772942
		fabric-particles-v1: fabric-particles-v1 0.2.2+4ea4772942
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.2.4+4ea4772942
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.1.2+4ea4772942
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.1.16+4ea4772942
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.3+4ea4772942
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.4+4ea4772942
		fabric-rendering-v0: Fabric Rendering (v0) 0.1.3+4ea4772942
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.1.5+4ea4772942
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 0.1.4+4ea4772942
		fabric-textures-v0: Fabric Textures (v0) 0.1.5+4ea4772942
		fabricloader: Fabric Loader 0.13.3
		java: Java HotSpot(TM) 64-Bit Server VM 8
		minecraft: Minecraft 1.14.4
		modmenu: Mod Menu 1.7.17+build.1
	Launched Version: Fabric 1.14.4
	LWJGL: 3.2.1 build 12
	OpenGL: Intel(R) UHD Graphics 617 GL version 2.1 INTEL-18.4.6, Intel Inc.
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because ARB_framebuffer_object is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Resource Packs: vanilla
	Current Language: English (US)
	CPU: 4x Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz
```

**Comment by zxcdevlop** [comment]
if this helps, this is whats going on for me: 
https://drive.google.com/file/d/1W3YJ87X-61h9vafv0axUlLXua0ORN2AP/view?usp=sharing

**Comment by jackassmc** [comment]
This appears to be https://github.com/TerraformersMC/ModMenu/issues/128 which was fixed in 1.16 and 1.15 but not 1.14. Deleting the Mod Menu config file should fix it.

**Comment by jackassmc** [comment]
@MagnusHJensen identified the root cause of this crash: `{"sorting": null}` in the config file. The same issue is present in 1.15.

> This appears to be #128 which was fixed in 1.16 and 1.15 but not 1.14.

It actually was fixed in 1.14, see https://github.com/TerraformersMC/ModMenu/commit/21e041501a76ca74b04014a7289ac0fa9b53fe2e.

**Comment by jackassmc** [comment]
Fixed in https://github.com/TerraformersMC/ModMenu/pull/474. Thanks @MagnusHJensen!
